web/management/commands/create_test_data.py

```
@@ -575,3 +604,257 @@ def handle(self, *args, **kwargs):
+                    question=question,
+                    text=f"Option {i + 1}",
+                    is_correct=(i == 0),
+                    order=i + 1,  # First option is correct
+                )
+        elif question.question_type == "true_false":
+            # Create true/false options
+            # always the True choice will be true
+            QuizOption.objects.create(question=question, text="True", is_correct=True, order=1)
+            QuizOption.objects.create(question=question, text="False", is_correct=False, order=2)
+
+    def generate_mock_answers(self, quiz: Quiz) -> dict:
```

Ruff (0.8.2)

`generate_mock_answers` is too complex (11 > 10)

(C901)